klippa/src/parsing_util.rs

```diff
@@ -65,6 +65,27 @@ pub fn parse_unicodes(unicode_str: &str) -> Result<IntSet<u32>, SubsetError> {
     Ok(result)
 }
 
+//parse input drop_tables string, which is a comma/whitespace-separated list of tables that will be dropped
+pub fn parse_drop_tables(input_str: &str) -> Result<IntSet<u32>, SubsetError> {
+    let mut result = IntSet::empty();
+
+    if input_str == "*" {
+        let out = IntSet::<u32>::all();
+        return Ok(out);
+    }
+
+    for tag in input_str.split(|c| c == ',' || c == ' ') {
+        if tag.is_empty() {
+            continue;
+        }
+        let Ok(table_tag) = Tag::new_checked(tag.as_bytes()) else {
+            return Err(SubsetError::InvalidTag(tag.to_owned()));
+        };
```

Comment:
this function returns an `InvalidTag` error describing the problem if the tag can't be parsed. Maybe we should hold on to that? (as in, have `SubsetError::InvalidTag { tag: String, error: InvalidTag }`)

Reply:
SubsetError::InvalidTag defined in lib.rs:
```
#[error("Invalid tag {0}")]
InvalidTag(String),
```